solc/CommandLineInterface.cpp

```diff
@@ -68,6 +69,7 @@
 	#include <unistd.h>
 #endif
 
+#include <numeric>
```

Comment:
Is this needed?

Reply:
Not anymore. Thanks for catching this. It's a leftover from an early version where I used `std::accumulate()` instead of `countEnabledOptions()`.